cmd/conduit/internal/testutils/mock_helpers.go

```diff
@@ -0,0 +1,144 @@
+// Copyright © 2025 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutils
+
+import (
+	"time"
+
+	"github.com/conduitio/conduit/cmd/conduit/api/mock"
+	apiv1 "github.com/conduitio/conduit/proto/api/v1"
+	"go.uber.org/mock/gomock"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func GetDateTime() *timestamppb.Timestamp {
```

Comment:
Maybe indicate in the method name this is the current time?

Reply:
@hariso It's not the current time! :P